packages/clients/dish/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - Fix: Extend typing for search result function according to type package update.
 - Fix: Import types `AddressSearchState` and `AddressSearchGetters` from correct position.
 - Fix: Import enum `SearchResultSymbols` from correct position.
+- Fix: Add translations for scale switcher.
```

Comment:
This changelog entry should be added to `@polar/plugin-scale`. Also, what do you think about "Add missing translations for the scale switcher."?

Reply:
ahhh I knew that >___< I must have mixed that up because of working on the dish client for too long.. 

Changed it: 48123d6a496b1519f9cc24ffd5ca296250ab0d56